sandbox/grommet-app/src/pages/layouts/kinds/Insider/Hero/Hero.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {
+    Box,
+    Grid,
+    Heading,
+    Paragraph,
+    Button
+} from 'grommet';
+import { LinkNext } from 'grommet-icons';
+
+export const Hero = ({...rest}) => {
+    return (
+    <Grid 
+        columns={['large', 'flex']}
+        gap="3xlarge"
+        pad={{ vertical: '4xlarge', horizontal: '4xlarge'}}
+        {...rest}
+    >
+        <Box 
+            align='start' 
+            gap="3xlarge"
+        >
+            <Box gap="xxlarge">
+                <Heading 
+                    level={1} 
+                    size="xxlarge" 
+                    margin="none"
+                >
+                    It starts with ambition.
```

Comment:
It seems like we need a heading flavor for the "condensed" display heading. Also, design wants this all caps. Should we align to start the discussion?